src/app/shared/components/menu/main-menu/main-menu.component.ts

```
@@ -162,5 +164,14 @@
     if (!this.toolTipText) {
       this.toolTipText = 'The integration will import all the newly updated ' + this.appName + ' dimensions and ' + brandingConfig.brandName + ' expenses in the configured state of export';
     }
+
+    if (this.router.url.includes("/token_expired/")){
+      this.isMenuDisabled = true;
+      this.modules = this.modules.map(item => ({
+        ...item,
+        disabled: item.disabled !== undefined ? item.disabled : this.isMenuDisabled
+      }));
+      
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)